### PR TITLE
fix: pages build but do not show

### PR DESCRIPTION
**Pull request by Dominik1999**
The playground homepage doesn't show, it can't find index.html. I think I fixed it now

### CODE REVIEW

File: playground/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "miden-assembly-examples-playground",
   "version": "0.1.0",
-  "homepage": "https://0xpolygonmiden.github.io/examples/",
+  "homepage": "https://0xpolygonmiden.github.io/examples/playground/",
```
Comment (grjte) [inline playground/package.json:4]: If this is being changed, then we should also update the playground link in the `README` (and anywhere else)

**Review by grjte — CHANGES_REQUESTED** [review]

File: playground/package.json (hunk 1 above)
Comment (Dominik1999) [reply playground/package.json:4]: Indeed. But I don't know if the change in the workflow will also change the URL. Somehow this is still a black box to me. Actually, it looks like the URL stays the same, `yourusername.github.io/your_repo_name` according to yourusername.github.io/your_repo_name

Comment (Dominik1999) [reply playground/package.json:4]: But to be honest, that is a bit tedious trial and error (for me)

**Review by grjte — APPROVED** [review]
Looks good, thanks!

**Pull request merged**